src/main/java/teammates/storage/api/StudentsDb.java

```diff
@@ -46,14 +47,15 @@ public void putDocument(StudentAttributes student){
         putDocument(Const.SearchIndex.STUDENT, new StudentSearchDocument(student));
     }
     
-    public StudentSearchResultBundle search(String queryString, String googleId, String cursorString){
+    public StudentSearchResultBundle search(String queryString, String cursorString,
+                                            List<InstructorAttributes> instructorRoles) {
```

Comment:
In some places, adding an extra parameter might make the method signature no longer self-explanatory. In those cases modify/add a header comment for the method.